src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -444,24 +467,152 @@ public void declaredLostEventIsPublished() {
 
   @Test
   public void cannotDeclareItemLostTwice() {
+    UUID servicePointId = servicePointsFixture.cd1().getId();
     final IndividualResource loan = declareItemLost();
     final UUID itemId = UUID.fromString(loan.getJson().getString("itemId"));
 
     assertThat(itemsClient.getById(itemId).getJson(), isDeclaredLost());
 
     final Response response = declareLostFixtures.attemptDeclareItemLost(
       new DeclareItemLostRequestBuilder()
+        .withServicePointId(servicePointId)
         .forLoanId(loan.getId()));
 
     assertThat(response.getJson(), hasErrorWith(allOf(
       hasMessage("The item is already declared lost"),
       hasParameter("itemId", itemId.toString()))));
   }
 
+  @Test 
+  public void shouldCancelUnpaidLostItemFeesWhenItemAgedToLostAndFeesAlreadyApplied() {
```

Comment:
Should this be declared lost instead of aged to lost?

Reply:
Yes, changing it now.

Reply:
Changing it now.